<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EEGoSportsSetupProjectWidget</class>
 <widget class="QWidget" name="EEGoSportsSetupProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Layout file:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="m_qLineEdit_EEGCap"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="m_qPushButton_EEGCap">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Cardinal points</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="m_comboBox_cardinalMode">
        <item>
         <property name="text">
          <string>Use tracked cardinals</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Use electrode shift</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="m_comboBox_LPA"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="m_label_Nasion">
        <property name="text">
         <string>Nasion:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="m_comboBox_RPA"/>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="m_doubleSpinBox_RPA">
        <property name="suffix">
         <string>cm/z</string>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="m_doubleSpinBox_Nasion">
        <property name="suffix">
         <string>cm/z</string>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="m_comboBox_Nasion"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="m_label_RPA">
        <property name="text">
         <string>RPA:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_label_cardinal">
        <property name="text">
         <string>Cardinal file:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_label_LPA">
        <property name="text">
         <string>LPA:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="m_doubleSpinBox_LPA">
        <property name="prefix">
         <string/>
        </property>
        <property name="suffix">
         <string>cm/z</string>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="m_pushButton_cardinalFile">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="m_lineEdit_cardinalFile"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_">
        <property name="text">
         <string>Mode:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
